.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - "1.8"
-  - "1.11"
+  - 1.8.x
```

Comment:
Since you mentioned the build fails on 1.8 due to murmur dependencies, do you think we should bump this to 1.9?

Reply:
Yes, I think so.